sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/SharedGalleryImageTests.java

```diff
@@ -213,6 +214,7 @@ public void canCreateUpdateGetDeleteGalleryImage() {
     }
 
     @Test
+    @Disabled
```

Comment:
shouldn't change this

Reply:
Fixes other than the network module have been rollbacked.